vars/rollingUpgradePipeline.groovy

```diff
@@ -170,6 +187,103 @@ def call(Map pipelineParams) {
                                                 }
                                             }
                                         }
+                                        stage('BYO Scylladb [optional]') {
+                                            catchError(stageResult: 'FAILURE') {
+                                                script {
+                                                    wrap([$class: 'BuildUser']) {
+                                                        dir('scylla-cluster-tests/scylla-pkg') {
```

Comment:
what's in `scylla-pkg` directory ? why not move to SCT top level dir as the rest of the places ?

Reply:
Because here we call remote project's pipeline which we do not control.
It may have it's own output files and so on.

Do you some problems having a sub-dir?

Reply:
It's just not exactly clear why, it looks like we might be cloning something, while we don't.

Reply:
also what do you mean with output files ? we are just calling a job, it's running on a different job, and gonna generate any files, and output files we need, we'll need to pull on our own from the job, don't we ?

Reply:
not critical